test/index.js

```diff
@@ -1,257 +1,245 @@
 'use strict';
+
+require('chai').should();
 const fs = require('fs');
 const render = require('../lib/renderer');
-const should = require('chai').should(); // eslint-disable-line
 const source = fs.readFileSync('./test/fixtures/markdownit.md', 'utf8');
+const { deepMerge } = require('hexo-util');
+const Hexo = require('hexo');
 
 describe('Hexo Renderer Markdown-it', () => {
+  const hexo = new Hexo(__dirname, { silent: true });
+  const parse = render.bind(hexo);
+  const defaultCfg = deepMerge({}, hexo.config);
+
+  beforeEach(() => {
+    hexo.config = deepMerge({}, defaultCfg);
```

Comment:
Is it possible to use `hexo.init()` or `hexo.load()` instead?

Reply:
`hexo.init()` works but there was a `MaxListenersExceededWarning` warning. Refactored to `JSON` functions.